db/revision_cache.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Number of recently-accessed doc revisions to cache in RAM
-const KDefaultRevisionCacheCapacity = 5000
+var KDefaultRevisionCacheCapacity uint32 = 5000
```

Comment:
The uint32/int gymnastics here (and below) feel a little awkward.  I think this is related to existing uint32 handling in config.go - can we just standardize this value as either int or uint32 everywhere?